### DISCUSSION TITLE
how to make file icon display in the widget area??

### DISCUSSION THREAD

**Discussion by zhoujingfighting** [eclipse-theia/theia#9705] [Q&A] [answered]
just now I found ,when I opened a file , I did not see the icon in the left side of title in the opened file widget area , but in Theia I can see the icon 
![image](https://user-images.githubusercontent.com/71758061/124569248-5cd77d00-de78-11eb-96a5-732f9e2d6c91.png)

so ,here comes my question 
where to config this ??? is this a default setting??when i click a file ,i want the icon showed in the opened widget!!

**Comment by vince-fugnitto** [comment, marked as answer]
@zhoujingfighting it is configurable through the `workbench.iconTheme` preference (similarly to vscode):

https://github.com/eclipse-theia/theia/blob/3bd43c98dc5c755de7ae23d3123a32e54096a4a1/packages/core/src/browser/core-preferences.ts#L66-L70

The default is `none`:

https://github.com/eclipse-theia/theia/blob/3bd43c98dc5c755de7ae23d3123a32e54096a4a1/dev-packages/application-package/src/application-props.ts#L68

**Comment by rcla** [comment]
In my case it has worked modifying the file 'settings.json'

Adding the line:
`"workbench.iconTheme": "theia-file-icons"`

**Reply by zhoujingfighting** [reply]
@rcla which `setting.json` file ????do I need to touch a new on in the root?

**Reply by rcla** [reply]
@zhoujingfighting

In the path of the current user's home directory. There is a ".theia" folder.

Modify the file "settings.son" If the file does not exist, you must create it.

Windows:
`C:\Users\username\.theia\settings.json`

Linux:
`/home/username/.theia/settings.json`
